docs/_user_guide/best_practices_archive.md

```diff
@@ -1,31 +1,35 @@
- - bundle all the files together with `tar` command
- - transfer the tarball off of yens
- - remove your project directory
- 
-Archiving data will save disk space on the yens as well as make it easy for you to transfer your full project as one big file.
+When your project is nearing completion and you have finished running computations on the Yens, what is left to do is:
+
+- Clean up files you no longer need.
+- Compress files with `zip` or `gunzip` command.
+- Bundle all the files together with the `tar` command.
+- Transfer the tarball off of Yens.
+- Remove your project directory.
 
-{% include warning.html content="The yens are not meant for long-term archiving of files and past projects. You should figure out where you want to archive your projects or talk to DARC team. Possible long term and cheap solutions include <a href=\"https://uit.stanford.edu/service/gsuite/drive\" target=\"_blank\">Google Drive</a>, <a href=\"https://aws.amazon.com/glacier/\" target=\"_blank\">AWS Glacier</a>, or <a href=\"https://code.stanford.edu/\" target=\"_blank\">GitLab</a>. You can store the archived files there and move them off of yens."%}
+Archiving data will save disk space on the Yens as well as make it easy for you to transfer your full project as one big file.
 
-{% include important.html content="Make sure you know risk classification of <a href=\"/security/index.html\" target=\"_blank\">the data you have on the Yens</a>. Some high risk data cannot be transferred off of the Yens to the cloud for example." %}
+!!! Warning
+    The Yens are **not** meant for long-term archiving of files and past projects. You should figure out where you want to archive your projects or talk to the DARC team. Possible long-term and affordable solutions include [Google Drive](https://uit.stanford.edu/service/gsuite/drive){:target="_blank"}, [AWS Glacier](https://aws.amazon.com/glacier/){:target="_blank"}, or [GitLab](https://code.stanford.edu/){:target="_blank"}. You can store the archived files there and move them off of Yens.
 
 ## How Do I Archive?
 
 ### Compressing Project Directory On Yen Servers
-Once you are done cleaning intermediate files and other junk from your project directory, you can compress the entire
-directory into one tarball that you can then push to Google Drive (with <a href="/faqs/rclone.html" target="_blank">rclone</a>) or other remote.
+Once you are done cleaning intermediate files and other junk from your project directory, you can compress the entire directory into one tarball that you can then push to Google Drive (with [rclone](/blog/2023/09/18/rclone-files-from-yens-to-google-drive/?h=rclone){:target="_blank"}) or other remote storage.
 
 If I have my project directory in `/zfs/projects/faculty/my-project-dir`, I would navigate to the parent directory above the directory I want to archive, then run:
 
-```bash
-$ cd /zfs/projects/faculty
-$ tar -zcvf my-project-name.tar.gz my-project-dir
+```title="Terminal Command"
```

Comment:
Disable the copy button by doing this:
{.yaml .no-copy title="Terminal Command"}

Reply:
This is the current content in the file:

```title="Terminal Command"
cd /zfs/projects/faculty
tar -zcvf my-project-name.tar.gz my-project-dir
```
I think .yaml .no-copy only applies to the output.